pkg/cluster/cluster.go

```diff
@@ -157,6 +158,12 @@ func New(ctx context.Context, log *logrus.Entry, _env env.Interface, db database
 		return nil, err
 	}
 
+	options := _env.Environment().EnvironmentCredentialOptions()
+	spTokenCredential, err := azidentity.NewEnvironmentCredential(options)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
`localFPAuthorizer` should be compatible with `fpCredRPTenant`.
I don't think we need to define a new credential.

Reply:
I think ```localFPAuthorizer``` uses autorest.Authorizer which is not an track2 sdk. They are doing simular jobs but since we are migrating to track2. we should stick with track2 SDK authorizer (azidentity)

Reply:
thanks @bitoku  ! i think it's working.